src/Protos/electricity-primitives.proto

```diff
@@ -15,9 +15,9 @@ message Commitment {
     bytes C = 1;
 }
 
-message TimePeriod {
-    google.protobuf.Timestamp DateTimeFrom = 1;
-    google.protobuf.Timestamp DateTimeTo = 2;
+message DateInterval {
+    google.protobuf.Timestamp start = 1;
```

Comment:
start with lower_case S -> maybe it should be Start for consistency purposes

Reply:
Good catch 👍

Reply:
Fixed